TuviSRPLib/TuviSRPLib/ProtonSRPServer.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using Org.BouncyCastle.Crypto;
+using Org.BouncyCastle.Crypto.Parameters;
+using Org.BouncyCastle.Math;
+using Org.BouncyCastle.Security;
+
+namespace TuviSRPLib
+{
+    /**
+	 * Implements the server side SRP-6a protocol. Note that this class is stateful, and therefore NOT threadsafe.
+	 * This implementation of SRP is based on the optimized message sequence put forth by Thomas Wu in the paper
+	 * "SRP-6: Improvements and Refinements to the Secure Remote Password Protocol, 2002"
+	 */
+    public class ProtonSRPServer
```

Comment:
Please, add a comment that this is an excerpt from BounsyCastle lib
Add the link to the original code